Alloy/commands/compile/parsers/Ti.UI.Button.js

```diff
@@ -36,6 +36,15 @@ function parse(node, state, args) {
 		node.setAttribute('systemButton', 'Ti.UI.' + iOSProxy + '.SystemButton.' + systemButton);
 	}
 
+	if ( node.hasAttribute('text') ) {
+		node.setAttribute('title', node.getAttribute('text'));
+		node.removeAttribute('text');
+	}
+
+	if ( node.getAttribute('verticalAlign') === 'center' ) {
+		node.setAttribute('verticalAlign', 'Titanium.UI.TEXT_VERTICAL_ALIGNMENT_CENTER');
```

Comment:
Based on [Josh's comment](https://jira.appcelerator.org/browse/ALOY-1703?focusedCommentId=450997&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-450997), this is a change in behaviour correct?

Reply:
The thing is, I am not talking about changing anything with the SDK.  This is simply an intuitive way to be able to center text and it mirrors what you can do with textAlign.  I think NOT doing this is actually more harmful as most developers are going to assume that "center" can be used and won't find out until their build breaks.

Reply:
But there is a behaviour change here, `verticalAlign: 'center'` currently does one thing (which I agree doesn't do what I would expect) and here we're changing it do something completely different.

Alloy has no special handling for `textAlign`, I think it's that that `TEXT_ALIGNMENT_CENTER` has a value of `center` so it just works. `TEXT_VERTICAL_ALIGNMENT_CENTER` has a value (on Android) of `middle` so it doesn't work.

If we're adding special handling for `center`, why not add the other variants?

Reply:
The thing is, no one knows about "middle" unless you dig into the source code.  "center" makes sense because it is part of the name TEXT_VERTICAL_ALIGNMENT_CENTER and the fact that TEXT_ALIGNMENT_CENTER can be aliased by using "center".   If we want to change the value to TEXT_VERTICAL_ALIGNMENT_MIDDLE, then "middle" would make sense.

Reply:
I wasn't saying people should know about middle, it was just an explanation of _why_ this weird behaviour happens.

I tried testing this change, and it looks like it only works when used in views and not in styles?